modules/tensor_mechanics/doc/content/source/auxkernels/RankTwoAux.md

```diff
@@ -37,13 +37,16 @@ AuxVariable is used as the argument for the `variable` input parameter in the `R
 
 !alert note title=Elemental vs Nodal Visualization of Quadrature Field Values
 Using an AuxVariable with `family = MONOMIAL` and `order = CONSTANT` will give a constant value of
-the AuxVariable for the entire element, which is computed by taking a volume-weighted average of the
-integration point quantities. Using an AuxVariable with `family = MONOMIAL` and `order = FIRST` or
-higher will result in fields that vary linearly (or with higher order) within each element, which are
-computed using a local least-squares procedure. Because the Exodus mesh format does not support
-higher-order elemental variables, these AuxVariables are output by libMesh as nodal variables for
-visualization purposes. Using higher order monomial variables in this way can produce smoother
-visualizations of results for a properly converged simulation.
+the AuxVariable for the entire element, which is computed by taking a local $L_2$ projection. Using an
```

Comment:
Is the original 'volume-weighted average' statement wrong? I think it is more generally understood.

Reply:
I haven't heard of it before... Anyways I have rolled it back to volume-weighted average.